executorlib/interactive/executor.py

```
@@ -166,14 +166,14 @@ def create_executor(
     backend: str = "local",
     max_cores: Optional[int] = None,
     cache_directory: Optional[str] = None,
-    resource_dict: Optional[dict] = None,
+    resource_dict: dict = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)